api/v1alpha1/packagebundle_test.go

```diff
@@ -34,6 +34,7 @@ func TestPackageBundle_Find(t *testing.T) {
 	expected := api.PackageOCISource{
 		Registry:   "public.ecr.aws/l0g8r8j6",
 		Repository: "eks-anywhere-test",
+		Name:       "v0.1.0",
```

Comment:
remove the v in front of 0.1.0 here I think

Reply:
The v in the version is a problem with the helm Chart.yaml and this is just test data anyway that isn't even used